spec/lib/rex/proto/kerberos/model/ticket_enc_part_spec.rb

```diff
@@ -0,0 +1,347 @@
+#                    @tag=2,
+#                    @tag_class=:UNIVERSAL,
+#                    @tagging=nil,
+#                    @value=#<OpenSSL::BN 1>>]>,
+#                #<OpenSSL::ASN1::ASN1Data:0x000055ae22e43f88
+#                 @indefinite_length=false,
+#                 @tag=1,
+#                 @tag_class=:CONTEXT_SPECIFIC,
+#                 @value=
+#                  [#<OpenSSL::ASN1::OctetString:0x000055ae22e43fb0
+#                    @indefinite_length=false,
+#                    @tag=4,
+#                    @tag_class=:UNIVERSAL,
+#                    @tagging=nil,
+#                    @value=
+#                     "0\x82\x02j0\x82\x02f\xA0\x04\x02\x02\x00\x80\xA1\x82\x02\\\x04\x82\x02X\x04\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\xBC\x01\x00\x00H\x00\x00\x00\x00\x00\x00\x00\n\x00\x00\x00\x1E\x00\x00\x00\b\x02\x00\x00\x00\x00\x00\x00\x06\x00\x00\x00\x14\x00\x00\x00(\x02\x00\x00\x00\x00\x00\x00\a\x00\x00\x00\x14\x00\x00\x00@\x02\x00\x00\x00\x00\x00\x00\x01\x10\b\x00\xCC\xCC\xCC\xCC\xAC\x01\x00\x00\xCC\xCC\xCC\xCCl\\\x00\x00\x80\x94\x11\f\x92\x84\xD8\x01\xFF\xFF\xFF\xFF\xFF\xFF\xFF\x7F\xFF\xFF\xFF\xFF\xFF\xFF\xFF\x7F\x80\x94\x11\f\x92\x84\xD8\x01\x00\x00\x00\x00\x00\x00\x00\x00\xFF\xFF\xFF\xFF\xFF\xFF\xFF\x7F\x14\x00\x14\x00\x97\xDF\x00\x00\x00\x00\x00\x00`\xE2\x00\x00\x00\x00\x00\x00\x84\x9D\x00\x00\x00\x00\x00\x00\x8C;\x00\x00\x00\x00\x00\x00\x94\xC2\x00\x00\x00\x00\x00\x00~\xC9\x00\x00\xF4\x01\x00\x00\xF4\x01\x00\x00\x01\x02\x00\x00\x05\x00\x00\x00r4\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x83\x88\x00\x00\x10\x00\x10\x00\x9C\x94\x00\x00\xA2\xD7\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x10\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\n\x00\x00\x00\x00\x00\x00\x00\n\x00\x00\x00f\x00a\x00k\x00e\x00_\x00m\x00y\x00s\x00q\x00l\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x05\x00\x00\x00\x01\x02\x00\x00\a\x00\x00\x00\x00\x02\x00\x00\a\x00\x00\x00\b\x02\x00\x00\a\x00\x00\x00\x06\x02\x00\x00\a\x00\x00\x00\a\x02\x00\x00\a\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\b\x00\x00\x00\x00\x00\x00\x00\b\x00\x00\x00D\x00W\x00.\x00L\x00O\x00C\x00A\x00L\x00\x05\x00\x00\x00\x01\x05\x00\x00\x00\x00\x00\x05\x15\x00\x00\x00\x02\x87ZU\xFDf{\xCC\xF7\xBC\xAF\x16\xF4\x01\x00\x00\x00\x00\x00\x00\x80\x94\x11\f\x92\x84\xD8\x01\x14\x00f\x00a\x00k\x00e\x00_\x00m\x00y\x00s\x00q\x00l\x00\x00\x00v\xFF\xFF\xFF\xE2X\xD7\xF5\x06\x17d\x9Cw\xC6\xFF\x16\x13\x16]\xE3\x00\x00\x00\x00v\xFF\xFF\xFF\xAFd\xEA\x1D\x9F\xA0c\xE9\xB1\x04\xD2S\xC4\x82\xCEa\x00\x00\x00\x00">]>]>]>]>]>]>
+# -*- coding:binary -*-
```

Comment:
Magic comments need to be at the top of the ruby file otherwise they do nothing; it might make sense to nuke the asn1 dump above